diff --git a/src/mainwindow.cpp b/src/mainwindow.cpp
index e288912..ca03c38 100644
--- a/src/mainwindow.cpp
+++ b/src/mainwindow.cpp
@@ -20,7 +20,7 @@ MainWindow::MainWindow(QWidget *parent) :
 
 
     // Init Database
-    QString     mNamaDb = "KBBI.db";
+    QString     mNamaDb = "/opt/KBBI-Qt/KBBI.db";
     QFileInfo   mFileDb(mNamaDb);
 
     // qDebug()<< "Lokasi DB : " << mFileDb.absoluteFilePath();
@@ -40,7 +40,7 @@ MainWindow::MainWindow(QWidget *parent) :
 
     // Connect to DB
     database = QSqlDatabase::addDatabase("QSQLITE");
-    database.setDatabaseName("KBBI.db");
+    database.setDatabaseName("/opt/KBBI-Qt/KBBI.db");
     database.open();
 
     // qDebug()<< "Is connect? " << database.open();
@@ -71,7 +71,7 @@ void MainWindow::copyDBfromRes(){
     // Nama File output copy
     // Tanpa full absolute path supaya diletakkan satu level
     // di direktori yang sama dengan file aplikasi
-    QString fileOut = "KBBI.db";
+    QString fileOut = "/opt/KBBI-Qt/KBBI.db";
 
     // Copy File dari Resource File
     if(QFile::copy(":/data/KBBI.db",fileOut)){
@@ -149,5 +149,3 @@ MainWindow::~MainWindow()
 {
     delete ui;
 }
-
-
